src/store.rs

```diff
@@ -480,7 +480,7 @@ impl<F: LurkField> Store<F> {
             let str_ptr = self.intern_str(s);
             ptr = self.intern_symcons(str_ptr, ptr);
         }
-        if sym.path == vec!["LURK", "NIL"] {
+        if sym.path == vec!["", "LURK", "NIL"] {
```

Comment:
At some point we want to make the root symbol just `vec![]` rather than `vec![""]`. I don't think it matters hugely whether we do this change in this PR, but if possible/easy to do so it might be better